env/production/locals.tf

```diff
@@ -23,6 +23,7 @@ locals {
     "rsv"             = ["rsv"],
     "seasonal-cov"    = ["seasonal-cov"],
     "seasonal-flu"    = ["seasonal-flu"],
+    "wnv"             = ["wnv"],
```

Comment:
Coming back here (after [an "ah ha" moment](https://bedfordlab.slack.com/archives/C03KWNCLK/p1727986323074139?thread_ts=1727985524.655479&cid=C03KWNCLK) in Slack) to note for posterity that this should have used `WNV`, as the actual repo name (e.g. https://github.com/nextstrain/WNV) and pathogen name (e.g. https://nextstrain.org/WNV and https://data.nextstrain.org/WNV_NA.json) are both capitalized. While GitHub's not case-sensitive when it comes to repo names, everything else involved is.